js/filter.js

```diff
@@ -53,6 +53,11 @@ const KEYWORDS = {
         const nextMonth = new Date(new Date(today.setMonth(month + 1)).setDate(1));
         const nextNextMonth = new Date(new Date(today.setMonth(month + 2)).setDate(1));
         return _buildKeywordQuery(field, operator, nextMonth, nextNextMonth);
+    },
+    "@recently-updated": () => {
+        const recentOrderDotSeconds = 600;
+        const recentTime = new Date((Date.now() - recentOrderDotSeconds * 1000));
+        return _buildKeywordQuery("last_updated", "gt", null, recentTime);

```

Comment:
This keyword can't be in ripe-commons-js because it relies on application-specific knowledge (the `last_updated` field and the `600`).

It would be an interesting addition if it can be added on Pulse, but not here.